packages/Search/test/tstLinearBVH.cpp

```diff
@@ -303,10 +303,10 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( LinearBVH, structured_grid, DeviceType )
     double Lx = 100.0;
     double Ly = 100.0;
     double Lz = 100.0;
-    int constexpr nx = 11;
-    int constexpr ny = 11;
-    int constexpr nz = 11;
-    int constexpr n = nx * ny * nz;
+    int nx = 11;
+    int ny = 11;
+    int nz = 11;
+    int n = nx * ny * nz;
```

Comment:
Is getting rid of `constexpr` necessary?

Reply:
> Necessary

No

We have two options:
* either we keep the const/constexpr declaration and we need an empty lambda capture
* or we get rid of it

Reply:
Sorry, I just don't know how things work. Can `constexpr` variables not be captured by lambdas?

Reply:
See commit message for 6ce897f